docs: hardening: Recommend systemd-sysctl service usage only initially (#2113)
rancher/rke2: docs/security/hardening_guide.md

sysctl -p /usr/local/share/rke2/rke2-cis-sysctl.conf
```

Please perform this step only on fresh installations, before actually using RKE2 to deploy Kubernetes. Many
Kubernetes components, including CNI plugins, are setting up their own sysctls. Restarting the

> `systemd-sysctl` service on a running Kubernetes cluster might cause bad effects.

Reviewer: ```suggestion
`systemd-sysctl` service on a running Kubernetes cluster can result in unexpected side-effects.
```
